gradebook.py

```diff
@@ -6,7 +6,7 @@ class Gradebook:
     def __init__(self):
         self.grades = {}  # {course_id: {student_id: {"grade": x, "timestamp": y}}}
 
-    def add_grade(self, instructor, course_id, student_id, grade):
+    def add_grade(self, instructor, course_id, student_id, grade, option = None):
```

Comment:
Please add a comment to explain what `option` is. What value `option` can have.

Reply:
After reading your code, I don't think you need to put the option in the argument.